app/(candidate)/onboarding/shared/ajaxActions.js

```diff
@@ -4,21 +4,12 @@ import gpApi from 'gpApi';
 import gpFetch from 'gpApi/gpFetch';
 import { deleteCookie } from 'helpers/cookieHelper';
 
-export async function updateCampaign(
-  campaign,
-  versionKey,
-  updateCandidate,
-  subSectionKey,
-) {
+export async function updateCampaign(keys, values) {
```

Comment:
Would love to see the signature for this method be just `updateCampaign(updates)` where updates is:
```javascript
{
  'key1.subkey1': value1,
  'key2.subkey2': value2,
  ...
}
```